client/src/components/ReviewAnswers.js

```diff
@@ -19,31 +96,56 @@ const ReviewAnswers = ({
   numOfQuestions,
 }) => {
   return (
-    <div>
+    <Container>
       <p>
         Your score is: {score} / {numOfQuestions}
       </p>
-      {questions.map(question => (
-        <Card key={question._id} className="mb-4">
-          <Card.Body>
-            <Card.Title>{question.text}</Card.Title>
-            <ul className="mb-0">
-              {question.answers.map(answer => (
-                <li
-                  key={answer._id}
-                  className={getAnswersClass(question, userAnswers, answer)}
-                >
-                  {answer.text}
-                </li>
-              ))}
-            </ul>
-          </Card.Body>
-        </Card>
-      ))}
-      <Button className="mb-4" variant="primary" href={quizName}>
-        Try again
-      </Button>
-    </div>
+      <Form>
+        <fieldset disabled>
```

Comment:
A fieldset is used for grouping inputs. Since there are no inputs inside it, it should be removed